doc/releases/release-notes-4.0.rst

```diff
@@ -166,6 +166,7 @@ Boards & SoC Support
     versions (1.0 and 1.1) are no longer supported.
   * Added ESP32 WROVER-E-N16R4 variant.
   * STM32H5: Added support for OpenOCD through STMicroelectronics OpenOCD fork.
+  * MAX32: Enabled Segger RTT and SystemView support.
 
 * Added support for these boards:
```

Comment:
Should max32690fthr be mentioned?

https://github.com/zephyrproject-rtos/zephyr/pull/77285

Reply:
is it not? https://github.com/zephyrproject-rtos/zephyr/blob/65511eacf9d17bf22a6cf28c3ace467a86a159f5/doc/releases/release-notes-4.0.rst?plain=1#L177

**all** boards added for this release have in fact already been contributed to the release notes in a previous commit (https://github.com/zephyrproject-rtos/zephyr/commit/12821f8ec6bb9d817a05fae1e5a51258c1d00d8e)

Reply:
is it not? https://github.com/zephyrproject-rtos/zephyr/blob/65511eacf9d17bf22a6cf28c3ace467a86a159f5/doc/releases/release-notes-4.0.rst?plain=1#L177

**all** boards added for this release have in fact already been contributed to the release notes in a previous commit (https://github.com/zephyrproject-rtos/zephyr/commit/12821f8ec6bb9d817a05fae1e5a51258c1d00d8e)

Reply:
Oh, I forgot fetch the latest changes. Sorry about that.